build/commands/scripts/commands.js

```diff
@@ -166,6 +167,19 @@ program
   .arguments('[build_config]')
   .action(build)
 
+program
```

Comment:
Can we add some comments explaining what this command does/how this works?

Reply:
done